website/content/docs/platform/k8s/injector/examples.mdx

```diff
@@ -385,3 +385,36 @@ spec:
 
 [pkiCert]: https://github.com/hashicorp/consul-template/blob/main/docs/templating-language.md#pkicert
 [writeToFile]: https://github.com/hashicorp/consul-template/blob/main/docs/templating-language.md#writeToFile
+
+## Cross namespace secret sharing ((#cross-namespace))
+
+1. Configure Vault for secret sharing across namespaces as described in the
+   [How-to configure cross namespace access in Vault Enterprise][cross-namespace]
+   Knowledge base article.
+1. Use the following Pod annotations to authenticate to the Kubernetes method in
```

Comment:
```suggestion
2. Use the following Pod annotations to authenticate to the Kubernetes method in
```
The render seems to fix it since it shows `2.` in the preview link.

Reply:
It's standard markdown notation to use `1.` for all entries in a numbered list. The renderer automatically sets the correct number and it means we don't need to worry about renumbering as the doc updates over time :)